### ISSUE TITLE
hugo-builder:0.55 → jguyomard/hugo-builder:0.55

### ISSUE THREAD

**Issue by sentience** [btholt/complete-intro-to-containers#9]
In the Introduction video, the README is shown and viewers are invited to pre-download all the docker containers needed for the course. Unfortunately, one of the container names was wrong. This was fixed in the README by https://github.com/btholt/complete-intro-to-containers/commit/a8ff678aa38e85599a1a69058165a5c0119e0e27; however, the course video still shows the wrong name (hugo-builder:0.55).

Is there any way to “patch” the video, or maybe include a pop-up correction of some kind? As it stands, it's an unfortunate speed bump in the first thing students are asked to do in the course.

**Comment by btholt** [comment]
Yeah, you're not wrong. It'd be cool if we could put an annotation there

cc @1Marc

**Comment by nikmk** [comment]
The annotation still does not exist about hugo-builder:0.55 @btholt
